loopin.go

```diff
@@ -999,6 +981,23 @@ func (s *loopInSwap) waitForSwapComplete(ctx context.Context,
 	return nil
 }
 
+// abandonSwapCancelInvoice abandons the swap and cancels the swap invoice.
```

Comment:
nit: either move in previous commit or don't move here to make the diff smaller and consistent with the commit intent.